socrates/lib/subscribers/subscriber.js

```diff
@@ -36,7 +36,15 @@ class Subscriber {
   }
 
   livesInGermany() {
-    return this.country() === 'DE';
+    return this.state.country && this.state.country.length > 0 && this.country() === 'DE';
```

Comment:
Das gleiche gilt hier. Die UI muss sicherstellen, dass dieses Feld gefüllt wird. Allerspätestens muss das in index.js abgefangen werden. Hier muss das Objekt immer sinnvoll befüllt sein.